specs/deneb/validator.md

```diff
@@ -98,6 +98,8 @@ All validator responsibilities remain unchanged other than those noted below.
 
 ##### Blob KZG commitments
 
+[New in Deneb:EIP4844]
```

Comment:
Seems like we should add more annotations to this doc -- e.g. to "Constructing the `SignedBlobSidecar`s" and the data structures added